trinity/protocol/les/normalizers.py

```diff
@@ -8,13 +8,22 @@
 from eth.rlp.headers import BlockHeader
 
 from trinity.protocol.common.normalizers import BaseNormalizer
+from trinity.rlp.block_body import BlockBody
 
 TResult = TypeVar('TResult')
 LESNormalizer = BaseNormalizer[Dict[str, Any], TResult]
 
 
+# Q: Shouldn't this be named GetBlockHeadersNormalizer?
 class BlockHeadersNormalizer(LESNormalizer[Tuple[BlockHeader, ...]]):
     @staticmethod
     def normalize_result(message: Dict[str, Any]) -> Tuple[BlockHeader, ...]:
         result = message['headers']
         return result
+
+
+class GetBlockBodiesNormalizer(LESNormalizer[Tuple[BlockBody, ...]]):
+    @staticmethod
+    def normalize_result(message: Dict[str, Any]) -> Tuple[BlockBody, ...]:
+        result = message['bodies']
+        return result
```

Comment:
I'm fairly sure that we'll want to return block body bundles here, similar to how the eth version does:
https://github.com/ethereum/py-evm/blob/9ba2c1fae5de83ae3fe55688197630e65262f1b9/trinity/protocol/eth/normalizers.py#L48-L57

but everywhere `msg` is in this snippet, using `result` from the PR.

This is going to change the type signature of the the exchange, and so will affect the validator, etc.